pkg/db/vulnerability_metadata.go

```diff
@@ -1,20 +1,30 @@
 package db
 
+// VendorMetadata captures non-standard CVSS fields that vendors can sometimes
+// include when providing CVSS information.  This vendor-specific metadata type
+// allows to capture that data for persisting into the database
+type VendorMetadata interface{}
+
 // VulnerabilityMetadata represents all vulnerability data that is not necessary to perform package-to-vulnerability matching.
 type VulnerabilityMetadata struct {
 	ID           string   // The identifier of the vulnerability or advisory
 	RecordSource string   // The source of the vulnerability information
 	Severity     string   // How severe the vulnerability is (valid values are defined by upstream sources currently)
 	Links        []string // URLs to get more information about the vulnerability or advisory
 	Description  string   // Description of the vulnerability
-	CvssV2       *Cvss    // Common Vulnerability Scoring System V2 values
-	CvssV3       *Cvss    // Common Vulnerability Scoring System V3 values
+	Cvss         []Cvss   // Common Vulnerability Scoring System values
 }
 
 // Cvss contains select Common Vulnerability Scoring System fields for a vulnerability.
 type Cvss struct {
+	VendorMetadata VendorMetadata
```

Comment:
It seems like the separate type definition isn't being used, so we could remove it and replace this line with:
```suggestion
	VendorMetadata interface{}
```